sdk/keyvault/keyvault-certificates/src/certificatesModels.ts

```diff
@@ -3,13 +3,30 @@ import { ParsedKeyVaultEntityIdentifier } from "./core/keyVaultBase";
 import {
   SecretProperties,
   CertificateAttributes,
+  CertificateOperation,
   KeyVaultClientCreateCertificateOptionalParams,
   JsonWebKeyType,
   JsonWebKeyCurveName,
   LifetimeAction,
   KeyUsageType
 } from "./core/models";
 
+export interface CertificatesClientInterface {
```

Comment:
I wish for a better name, but am having trouble thinking of a good one... this is why I used to do the `I` prefix thing that nobody likes anymore 😅